selfdrive/car/gm/carcontroller.py

```diff
@@ -176,10 +176,17 @@ def update(self, sendcan, enabled, CS, frame, actuators, \
     if self.lkas_active and CS.lkas_status != 1:
       # SCM ignores steering command. Workaround is to
       # temporary disable steering command.
-      # Re-enabling after 200ms seems to be working great
+      # Re-enabling after 1 second works better.
       self.lkas_active = False
       if self.inhibit_steer_for == 0:
-        self.inhibit_steer_for = 20
```

Comment:
It should throw an alert while this is happening

Reply:
Steering temp unavail is presented until recovery is completed